watsong/webapp.py

```diff
@@ -36,4 +35,13 @@ def index() -> Any:
         """
         return redirect(url_for("jukebox.jukebox"))
 
+    app.config.update(
+        # SESSION_COOKIE_SECURE=True,
+        # SESSION_COOKIE_HTTPONLY=True,
+        SESSION_COOKIE_SAMESITE="Lax",
+        SESSION_TYPE="filesystem",
+    )
+
+    Session(app)
```

Comment:
I'm not sure what these lines of code do

Reply:
Vanilla flask uses browser cookies to store session information. Since we store songs in the session (maybe not a good architectural decision), we reach the cookie storage limit. flask_session is a flask extension to support server-side sessions. These lines set the samesite attribute for cookies (recommended by Mozilla [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite)) and tell flask_session to use the filesystem for server-side sessions, rather than software like redis or memcached.